godot-core/src/builtin/color.rs

```diff
@@ -38,23 +38,9 @@ pub struct Color {
     pub a: f32,
 }
 
-impl Color {
-    // TODO implement all the other color constants using code generation
-
-    /// Transparent black.
-    pub const TRANSPARENT_BLACK: Color = Self::from_rgba(0.0, 0.0, 0.0, 0.0);
-
-    /// Transparent white.
-    ///
-    /// _Godot equivalent: `Color.TRANSPARENT`_
-    pub const TRANSPARENT_WHITE: Color = Self::from_rgba(1.0, 1.0, 1.0, 0.0);
-
-    /// Opaque black.
-    pub const BLACK: Color = Self::from_rgba(0.0, 0.0, 0.0, 1.0);
-
-    /// Opaque white.
-    pub const WHITE: Color = Self::from_rgba(1.0, 1.0, 1.0, 1.0);
+include! {"color_constants.rs"}
```

Comment:
I think that's the more common usage, also suggested [by docs](https://doc.rust-lang.org/std/macro.include.html):
```suggestion
include!("color_constants.rs");
```

But I think here we don't even need `include`, a regular module would also work?
With the `mod color_constants;` being declared in `mod.rs`...

Not sure if we can influence this, but if somehow possible, it would be nice to list the constants _after_ the methods in the docs, so that people see the more common used information on top. If needed, we can resort to `include!` but after the other impl.

You can use the bot's link as a check for the docs, it will be updated with a latency of a few minutes on each push 🙂